media-tv/plex-media-server/plex-media-server-1.23.4.4805.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1 systemd unpacker
+
+MY_PV="${PV}-186bae04e"
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+
+DESCRIPTION="Free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+S="${WORKDIR}"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist"
+
+DEPEND="
+	acct-group/plex
+	acct-user/plex"
+
```

Comment:
Seem to have lost `RDEPEND` entirely by removing openssl, still need `RDEPEND="${DEPEND}"` for those acct-* at runtime.

On a side-note, maybe add a remove old commit too? I don't think need 4 versions at once, keeping 2 should be enough.

Reply:
Fixed, thanks :-)